### ISSUE TITLE
Stripe Support

### ISSUE THREAD

**Issue by Geenz** [Yellow-Dog-Man/Resonite-Issues#2819]
### Is your feature request related to a problem? Please describe.

Presently, our main payment processor is Patreon.  Patreon takes a high fee for our usage - averaging around 14%.  We mostly use this as a payment processor and subscription gateway.

There are much cheaper alternatives that also provide a much wider array of payment options, such as Stripe.

### Describe the solution you'd like

Stripe accepts payments globally at a much cheaper rate, and also provides options for far more integrated subscription management and billing.  We should:
- Add support for Stripe Checkout
- Add support for Stripe Billing
- Add support for subscriptions via Stripe Billing

### Describe alternatives you've considered

Continuing to pay Patreon ~14%.

**Comment by gentlecolts** [comment]
Will we see #520 as part of this?

**Comment by Geenz** [comment]
That's a different issue.

**Comment by Mhowser** [comment]
Has there been any progress on this? I have been wanting to support you guys for a while but unable to use Patreon.

**Comment by troyBORG** [comment]
Yeah they talked about it in the January 2025 Recap Stream....
https://www.twitch.tv/videos/2343535503?t=0h41m52s

**Comment by Frooxius** [comment]
Yes. We're currently testing this in closed beta and it should be open soon (within next few weeks) once the blocker issues are resolved!

**Comment by ColinTimBarndt** [comment]
Can this issue be closed now that Stripe can be used by everyone?
